README.md

```diff
@@ -87,9 +87,8 @@ Precompiled actor bundles are provided as [release binaries][releases] in this r
 We release all actors, the runtime, and the state abstraction at the same time by:
 
 1. Changing the `workspace.package.version` in the top-level `Cargo.toml` file.
-2. Creating a [release][releases] in GitHub.
 
-This will trigger an automatic bundle-build by GitHub CI, and the generated bundles will be attached to the GitHub release.
+This will trigger an automatic GitHub release and associated tags creation and then it will also trigger bundle-build by GitHub CI, and the generated bundles will be attached to the GitHub release.
```

Comment:
```suggestion
This triggers CI to create a draft GitHub Release, create the associated tags, and built and upload assets to the release.  When the PR is merged, the GitHub Release will be published.
```

I'm not sure if I have the terminology right, but I think that's encompassing of the overall flow